kratos/sources/kratos_parameters.cpp

```diff
@@ -483,6 +483,24 @@ bool Parameters::RemoveValue(const std::string& rEntry)
 /***********************************************************************************/
 /***********************************************************************************/
 
+bool Parameters::RemoveValues(const std::vector<std::string>& rEntries)
+{
+    bool sucess = true;
+    for (auto& r_entry : rEntries) {
+        sucess = this->Has(r_entry);
+        if (!sucess) break;
+    }
+    if (sucess) {
+        for (auto& r_entry : rEntries) {
+            this->RemoveValue(r_entry);
+        }
+    }
+    return sucess;
```

Comment:
```suggestion
    for (const auto& r_entry : rEntries) {
        if (!this->Has(r_entry)) return false;
    }
    for (const auto& r_entry : rEntries) {
            this->RemoveValue(r_entry);
    }
    return true;
```

`const` + don't need the extra `bool`